apps/ios/GuideDogs/Code/Data/Authored Activities/AuthoredActivityMetadata.swift

```diff
@@ -45,20 +45,18 @@ struct AuthoredActivityMetadata: Codable, CustomStringConvertible {
     }
     
     /// Builds the remote server path that the content can be downloaded from
-    /// E.G. https://share.openscape.io/experiences/some-activity.gpx
+    /// E.G. https://share.soundscape.services/experiences/<id>/activity.gpx
     var downloadPath: URL? {
-        var components = URLComponents()
+    var components = URLComponents()
         
         switch linkVersion {
-        case .v1:
-            components.scheme = "https"
-            components.host = "share.openscape.io"
-            components.path = "/experiences/\(id).gpx"
         case .v2, .v3:
-            // Version 2 and 3 links also look the same (perk of forking)
             components.scheme = "https"
             components.host = "share.soundscape.services"
-            components.path = "experiences/\(id).gpx"
+            components.path = "/activities/\(id)/activity.gpx"
+        default:
+            // no other versions currently supported
+            break
```

Comment:
I think all cases were handled explicitly before. Swift does require all cases to be handled either explicitly or with a default case.

I was thinking of proposing a v4 link in the future that would contain the full URL to the activity.gpx file. That would allow the authoring tool to be used on other domains for development etc.